kotlin-android/app/src/main/java/com/mongodb/tasktracker/TaskActivity.kt

```diff
@@ -43,8 +43,13 @@ class TaskActivity : AppCompatActivity() {
             startActivity(Intent(this, LoginActivity::class.java))
         }
         else {
-            // configure realm to use the current user and the partition corresponding to "My Project"
-            val config = SyncConfiguration.Builder(user!!, "My Project")
+            // get the partition value and name of the project we are currently viewing
+            val partition = intent.extras?.getString(PARTITION_EXTRA_KEY)
+            val projectName = intent.extras?.getString(PROJECT_NAME_EXTRA_KEY)
+
+            // display the name of the project in the action bar
+            title = projectName
```

Comment:
Random Kotlin question: is `title` a global? Also, do you need to define a variable for `projectName` or could you assign the title directly from the intent?

Reply:
Yeah, this has to do with Kotlin-isms that made sense for java but are a little less obvious in Kotlin. Basically we're extending the Activity class, which contains a ``title`` member variable. In Java, you would very explicitly setTitle(newTitle), but Kotlin does away with explicit getters and setters, so you just get ``title = ...``. I'll add more to the comment to explain this since it is *very* unintuitive.